plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -1812,6 +1819,7 @@ protected void testTargetMaxFileSize(int expectedTableWriters)
         Session session = Session.builder(getSession())
                 .setSystemProperty("task_writer_count", "1")
                 .setSystemProperty("scale_writers", "false")
+                .setSystemProperty("task_scale_writers", "false")
```

Comment:
There is also testTargetMaxFileSizePartitioned and similar tests in iceberg and delta
Check for all tests where task_writer_count is explicitly set